### PR TITLE
Hotfix issue with selecting empty accounts when HW scanning

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: src/features/AddAccount/components/HDWList.tsx [hunk 1]
```diff
@@ -151,25 +151,18 @@ export default function HDWList({
   };
 
   const handleSelection = (account: TableAccountDisplay) => {
```
Comment (Mrtenz) [inline src/features/AddAccount/components/HDWList.tsx:153]: Can you test this?

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/features/AddAccount/components/HDWList.tsx (hunk 1 above)
Comment (FrederikBolding) [reply src/features/AddAccount/components/HDWList.tsx:153]: Great idea!

File: src/features/AddAccount/components/HDWTable.tsx [hunk 2]
```diff
@@ -337,7 +337,12 @@ const HDTable = ({
       ) : (
         <Body>
           {accountsToDisplay.map((account: TableAccountDisplay, index) => (
-            <Row key={index} onClick={() => onSelect(account)} isSelected={account.isSelected}>
+            <Row
+              key={index}
+              onClick={() => onSelect(account)}
```
Comment (Mrtenz) [inline src/features/AddAccount/components/HDWTable.tsx:342]: Don't use arrow functions in props

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Review by Mrtenz — APPROVED** [review]

File: src/features/AddAccount/components/HDWTable.tsx (hunk 2 above)
Comment (FrederikBolding) [reply src/features/AddAccount/components/HDWTable.tsx:342]: Added card to fix this in another PR

**Pull request merged**